### PR TITLE
allow default shared secret to be set on github gateway

### PR DESCRIPTION
**Pull request by krancour**
Complements https://github.com/brigadecore/brigade-github-app/pull/54

__Edit:__ _After_ we do the next release of this gw chart, I want to make the `brigade` chart depend on that latest release and then surface this new setting there as well.

### CODE REVIEW

File: charts/brigade-github-app/values.yaml [hunk 1]
```diff
@@ -52,3 +52,4 @@ github:
   # Trigger a Check Suite on Pull Requests
   # This will need to be set to true to enable running Check Suites on PRs originating from forks
   checkSuiteOnPR: true
+  # defaultSharedSecret:
```
Comment (vdice) [inline charts/brigade-github-app/values.yaml:58]: I'm wondering if we can add a further line of context above?  Something describing how this value can be set to the shared webhook secret set on the corresponding GitHub App and that this value will be inherited/used by any/all Brigade projects it serves, if not set on the project-level... (assuming I have that right 😄 )

**Review by vdice — APPROVED** [review]
LGTM, though one suggestion for comment/context.

File: charts/brigade-github-app/values.yaml (hunk 1 above)
Comment (krancour) [reply charts/brigade-github-app/values.yaml:58]: Sure. On it.

Comment (krancour) [reply charts/brigade-github-app/values.yaml:58]: Added it in the latest commit.

**Review by vdice — APPROVED** [review]

**Pull request merged**